gateway-provider-security-pac4j/src/main/java/org/apache/knox/gateway/pac4j/filter/Pac4jDispatcherFilter.java

```diff
@@ -214,6 +214,18 @@ public void init( FilterConfig filterConfig ) throws ServletException {
 
   }
 
+  private String resolveAlias(String clusterName, String key, String value) throws ServletException {
+    if (value.startsWith("${ALIAS=") && value.endsWith("}")) {
+      String alias = value.substring("${ALIAS=".length());
```

Comment:
Why not remove the end delimiter here instead of performing another substring subsequently?

Reply:
@pzampino good idea, fixed it.

Reply:
`String alias = value.substring(8, alias.lenght()-1);`